cypress/integration/header.spec.ts

```diff
@@ -1,57 +1,40 @@
 /// <reference path="../support/index.d.ts" />
```

Comment:
Not that it's your change but do we know what this is doing?

Reply:
Telling cypress to use TS